### PR TITLE
Fixing the feedback message error

### PR DESCRIPTION
**Pull request by mosesmulumba**
# Description

This is to  help eliminate the error on the feedback message after searching for the projects.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Trello Ticket ID

https://trello.com/c/EkVge1Io

## How Can This Been Tested?

Pull the remote branch and run it locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

before
before

after
after

### CODE REVIEW

**Review by neelxie — COMMENTED** [review]
Add screenshots sir @mosesmulumba

File: src/components/AppsList/index.jsx [hunk 1]
```diff
@@ -101,6 +97,7 @@ class AppsList extends Component {
     );
     return (
       <div >
+        {console.log(allApps)}
```
Comment (jnsereko) [inline src/components/AppsList/index.jsx:100]: is there any reason why you loging this?

Comment (khalifan-kfan) [reply src/components/AppsList/index.jsx:100]: nice cache @jnsereko

**Review by neelxie — DISMISSED** [review]
LGTM

**Pull request merged**